Gemfile

```diff
@@ -1,31 +1,40 @@
 source 'https://rubygems.org'
 
+# Is this being used? If so, please add comment.
 git_source(:github) do |repo_name|
   repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
   "https://github.com/#{repo_name}.git"
 end
```

Comment:
Is this a default part of the Gemfile?

Reply:
Im guessing it is, I didn't add it. Maybe its new for rails 5?

Reply:
Thanks. I see it is explained here:

- https://stackoverflow.com/a/41454572/6763239

I'll update the comment.